bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx

```diff
@@ -43,10 +45,12 @@ export default class ActionsBar extends Component {
     );
   }
 
-  renderForUser() {
+  renderForUser(isVoiceUser) {
```

Comment:
I know this was not introduced in this PR but I couldn't help but notice that `renderForUser` and `renderForPresenter` contain almost the same code except for some lines (a div and it's wrapped componend and the style classes). They could be consolidated into only one method.